single_upload.py

```diff
@@ -69,7 +59,37 @@
     inquirer.List(
         "option", 
         message="Select the media category and press return", 
-        choices=["Automotive","Books and Literature","Business and Finance","Careers","Education","Events and Attractions","Family and Relationships","Fine Art","Food and Drink","Healthy Living","Hobbies and Interests","Home and Garden","Medical Health","Movies","Music and Audio","News and Politics","Personal Finance","Pets","Pop Culture","Real Estate","Religion and Spirituality","Science","Shopping","Sports","Style and Fashion","Technology and Computing","Television","Travel","Video Gaming"],
```

Comment:
Existing line of code went on too long. Different teams will have different line limits (MPD at JW has 120 character line limits), but in general whenever I have to scroll to read a line in VS Code, I start including line breaks.